src/PLTPlane.cc

```diff
@@ -348,6 +348,15 @@ bool PLTPlane::IsFiducial (FiducialRegion const FidR, int const Col, int const R
       }
       return false;
       break;
+    case kFiducialRegion_FullSensor:
```

Comment:
As I mentioned earlier, it might be nice to print out a warning if the pixel falls outside the whole sensor region (since that really shouldn't happen normally). Your choice though.